Source/MediaInfo/HashWrapper.h

```diff
@@ -49,7 +50,7 @@ class HashWrapper
         SHA512,
         HashFunction_Max,
     };
-    typedef bitset<HashFunction_Max> HashFunctions;
+    typedef std::bitset<HashFunction_Max> HashFunctions;

```

Comment:
CI - error https://travis-ci.org/MediaArea/MediaInfoLib/jobs/170764480
../../../Source/MediaInfo/Multiple/File_Mxf.h:734:5: error: ‘bitset’ does not name a type

     bitset<Stream_Max+1> StreamPos_StartAtZero; //information about the base of StreamPos (0 or 1, 1 is found in 1 file) TODO: per Essence code (last 4 bytes of the Essence header 0xTTXXTTXX)

     ^